config/config.py

```diff
@@ -15,11 +15,12 @@
                      "NSTHNSTHaoensutCGSRCGnsthoesucgsrSNTH")
 GOOGLE_ANALYTICS_ID = 'UA-10305579-1'
 
+DATABASE_SERVER_NAME = env.get('DATABASE_SERVER_NAME', 'localhost')
 SQLALCHEMY_DATABASE_URI = env.get(
     'SQLALCHEMY_DATABASE_URI',
```

Comment:
instead of templating the hostname in, let's just use SQLALCHEMY_DATABASE_URI which is already defined

Reply:
Okay